src/routes.ts

```diff
@@ -1,8 +1,8 @@
 export const ROUTES = {
```

Comment:
Tohle je je spravne podle toho, jak jsme to spolecne delali, ale bude potreba to zrefaktorovat trosku at je to jednodussi na použití a čtení

Reply:
Koukala jsem na to a upřímně nevím jak to zjednodušit, uděláme to pak spolu ?

Reply:
Urcite :)